src/table/hooks/use-column-widths.ts

```diff
@@ -13,6 +13,14 @@ import useOnPropsChange from '../virtualized-table/hooks/use-on-props-change';
 
 type GetHugWidth = (headLabel: string, totalsLabel: string, glyphCount: number) => number;
 
+const HEADER_CELL_BUTTON_PADDING = 8 * 2;
```

Comment:
These constants are a bit awkward as the number seem to be MUI defaults or from `muiTheme.space()`. So I have not really found a good way yet to resolve them dynamically.